frame/society/src/lib.rs

```diff
@@ -552,8 +713,8 @@ decl_module! {
 		///
 		/// Total Complexity: O(M + B + C + logM + logB + X)
 		/// # </weight>
-		#[weight = T::BlockWeights::get().max_block / 10]
-		pub fn bid(origin, value: BalanceOf<T, I>) -> DispatchResult {
+		#[pallet::weight(T::BlockWeights::get().max_block / 10)]
```

Comment:
Out of scope of this PR, but this weight seems strange - anyone know if this should be properly benchmarked?

Reply:
yes it should be done, but the weight is probably an upper bound for now, at least for our usage in kusama I guess, so maybe not the priority